cif2cell/cif2cell.xml

```diff
@@ -16,12 +16,99 @@
+                #if $cartesian_conditional.cartesian == "true":
+                    file_name=\$(sed 's/ //g' <<< '$file.name') &&
+                    ln -s '$file' \$file_name &&
+                    cif2cell -f \$file_name -p castep -o out.cell --no-reduce --vca --cartesian
+                #end if
+            #end if
+        #end if
+        #if $use_no_reduce_conditional.use_no_reduce == "true":
+            #if $vca_conditional.vca == "true":
+                #if $cartesian_conditional.cartesian == "false":
+                    file_name=\$(sed 's/ //g' <<< '$file.name') &&
+                    ln -s '$file' \$file_name &&
+                    cif2cell -f \$file_name -p castep -o out.cell --no-reduce --vca
+                #end if
+            #end if
+        #end if
+        #if $use_no_reduce_conditional.use_no_reduce == "true":
+            #if $vca_conditional.vca == "false":
+                #if $cartesian_conditional.cartesian == "true":
+                    file_name=\$(sed 's/ //g' <<< '$file.name') &&
+                    ln -s '$file' \$file_name &&
+                    cif2cell -f \$file_name -p castep -o out.cell --no-reduce --cartesian
+                #end if
+            #end if
+        #end if
+        #if $use_no_reduce_conditional.use_no_reduce == "false":
+            #if $vca_conditional.vca == "true":
+                #if $cartesian_conditional.cartesian == "true":
+                    file_name=\$(sed 's/ //g' <<< '$file.name') &&
+                    ln -s '$file' \$file_name &&
+                    cif2cell -f \$file_name -p castep -o out.cell --vca --cartesian
+                #end if
+            #end if
+        #end if
+        #if $use_no_reduce_conditional.use_no_reduce == "false":
+            #if $vca_conditional.vca == "false":
+                #if $cartesian_conditional.cartesian == "true":
+                    file_name=\$(sed 's/ //g' <<< '$file.name') &&
+                    ln -s '$file' \$file_name &&
+                    cif2cell -f \$file_name -p castep -o out.cell --cartesian
+                #end if
+            #end if
+        #end if
+        #if $use_no_reduce_conditional.use_no_reduce == "false":
+            #if $vca_conditional.vca == "true":
+                #if $cartesian_conditional.cartesian == "false":
+                    file_name=\$(sed 's/ //g' <<< '$file.name') &&
+                    ln -s '$file' \$file_name &&
+                    cif2cell -f \$file_name -p castep -o out.cell --vca
+                #end if
+            #end if
+        #end if
     ]]></command>
     <inputs>
         <param type="data" argument="--file" format="cif" label="Structure file to Convert (.cif)" />
+        <conditional name="use_no_reduce_conditional">
+            <param name="use_no_reduce" type="select" display="radio" value="false" label="Do you want to construct a primitive cell using --no-reduce?" help="Use a cif file to convert into a structural (cell) file.">
```

Comment:
We can use `argument` instead of `name`:
https://galaxy-iuc-standards.readthedocs.io/en/latest/best_practices/tool_xml.html#parameter-name-argument-and-help
i.e. `argument="--no-reduce"`